docs/source/ucx_features.rst

```diff
@@ -57,6 +57,6 @@ Protocols, Optimizations and Advanced Features
 - Pipeline protocols for GPU memory
 - QoS and traffic isolation for RDMA transports
 - Platform (micro-architecture) specific optimizations (such as memcpy, memory barriers, etc.)
-- Multi-rail support
+- Multi-rail and RoCE LAG support
```

Comment:
`Multi-rail and RoCE LAG support` -> `Multi-rail and line aggregation group (ROCE) support`

Reply:
@shamisp Done